CMakeLists.txt

```diff
@@ -110,7 +154,9 @@ if(NOT "${HAVE_QRCODEGEN_H}")
 endif()
 set_property(GLOBAL APPEND PROPERTY PACKAGES_FOUND qrcodegen)
 endif()
-find_library(LIBRT rt)
+if(NOT APPLE)
```

Comment:
i wonder if we ought just check for librt, and use it iff found? i don't think we'll run into many cases where it's necessary but not present, as it's part of libc. it's not going to be available on e.g. windows either.

Reply:
It would be simpler, definitely